BUILDING

```diff
@@ -210,10 +210,10 @@ env OS=Windows_NT make
 Prerequisites:
 
 * Bash/WSL, MinGW/MSYS, or Cygwin with bash, git, grep, make, sed, etc.
-* Microsoft Visual Studio 2017 or 2015
+* Microsoft Visual Studio 2019, 2017 or, 2015
```

Comment:
Please move the last comma to be immediately after 2017: Microsoft Visual Studio 2019, 2017, or 2015